packages/block-editor/src/components/block-list/style.scss

```diff
@@ -319,6 +319,11 @@
 	}
 }
 
+.is-root-container > .alignfull {
+	margin-right: calc(var(--wp--style--root--padding-right) * -1);
+	margin-left: calc(var(--wp--style--root--padding-left) * -1);
+}
```

Comment:
This is where it gets really tricky to handle shorthand padding values. Perhaps we can add this rule in dynamically somewhere else.

Reply:
If I'm not wrong it's not possible to have "align full" in the site editor (or any editor) unless there's a layout defined in the root which only happens in the post editor if the theme.json has "layout" key right? Makes me wonder if this should be moved to the edit-post or something like that to explain its purpose (maybe add a comment)